Epic/unit level orders

MarcinGanczorz opened the pull request:
Feature Description on wiki: https://wiki.one.int.sap/wiki/display/CXCOMPGMRM/Display+unit-level+orders
Jira Ticket: https://jira.tools.sap/browse/CXSPA-1298

I539321 committed:
cms changes for a new link and component

I539321 committed:
changes to the unit-level component

I539321 committed:
working orders and unit-level orders components

I539321 committed:
clean up dead code and add styling

I539321 committed:
drop unused selector and methods

I539321 committed:
adapt to the new Commerce migration changes

I539321 committed:
drop the OrderUser interface

I539321 committed:
styling changes

MarcinGanczorz committed:
add missing style classes; adjust table column width

MarcinGanczorz committed:
Merge branch 'feature/CX-15439-cms-styling' into epic/unit-level-orders

MarcinGanczorz committed:
refactor: move unit-level orders feature to organization library (#16101)

* move unit-order feature to organization library

* add unit-order feature to storefrontapp

* add missing unit tests

Co-authored-by: Andi Braimllari <andi.braimllari@sap.com>

MarcinGanczorz committed:
Merge remote-tracking branch 'origin/develop' into epic/unit-level-orders

# Conflicts:
#	feature-libs/order/assets/translations/en/order.i18n.ts
#	feature-libs/order/root/model/order.model.ts

I565452 committed:
CXEC-15457: add fields to order details page for buyer and unit (#16077)

changes are approved as part of the approval for the CXEC-17290  (https://github.com/SAP/spartacus/pull/16126)

I565452 committed:
CXEC-17289: add e-mail address to order details page (#16117)

changes are approved as part of the approval for the CXEC-17290 (https://github.com/SAP/spartacus/pull/16126)

I565452 committed:
CXEC-17290: unit-level orders - add e2e tests for order details page (#16126)

MarcinGanczorz committed:
Merge branch 'develop' into epic/unit-level-orders

MarcinGanczorz committed:
CXSPA-1500: upgrade to Angular 14

MarcinGanczorz committed:
CXSPA-1469_CXSPA-1471: add sort translations (#16278)

MarcinGanczorz committed:
Feature/CXSPA-1462 remove component header (#16279)

* CXSPA-1462: remove component header

* CXSPA-1462: remove test for component header

I565452 committed:
feature/cxspa-1474 add email address to the unit-level order history list (#16275)

* CXSPA-1474: add email address to the unit-level order history list

* CXSPA-1474: fix email in detail view

* CXSPA-1474: fix prettier and dependencies issues

* CXSPA-1474: fix missing email field in the B2BUser object definition

* CXSPA-1474: fix missing email field in the unit test

MarcinGanczorz committed:
CXSPA-1461: update no orders text message (#16305)

I565452 committed:
CXSPA-1474: update the color for email field (#16328)

T0bas-2 committed:
Feature/CXSPA-1350: Implement user rights assignment functionality for unitorderviewergroup (#16274)

* CXSPA-1350: Create rights form section and checkboxes

* CXSPA-1350: Add rights information to user details view

* CXSPA-1350: Add rights section to popup window

* CXSPA-1350: Change unit/user form to display rights

* CXSPA-1350: Fix unit tests

* CXSPA-1350: Refactor unitorderviewergroup role to right

* CXSPA-1350: Remove rights from role column

* CXSPA-1350: Resolve multiple selection conflict

* CXSPA-1350: Add additional e2e test case for user rights

MarcinGanczorz committed:
CXSPA-1463: add total result to unit-level orders history (#16350)

* CXSPA-1463: add total result to unit-level orders history

* CXSPA-1463: fix stylelint issues

* CXSPA-1463: small refactoring

MarcinGanczorz committed:
Merge branch 'develop' into epic/unit-level-orders

# Conflicts:
#	feature-libs/organization/README.md
#	feature-libs/organization/_index.scss

github-actions[bot] committed:
Add license header

giancorderoortiz committed:
Merge branch 'develop' into epic/unit-level-orders

I565452 committed:
CXSPA-1614: update color for email field in order history list (#16410)

T0bas-2 committed:
Feature/CXSPA-1665: Hide user rights column if no rights assigned (#16421)

I565452 committed:
Feature/cxspa 1615 1688 (#16436)

* CXSPA-1615: unit-level order history - correct vertical spacing of elements

* CXSPA-1688: unit-level order history - fix date field is not a link to the details page

T0bas-2 committed:
Feature/CXSPA-1611: Truncate email address with ellipsis in unit-level order list (#16422)

* CXSPA-1611: Truncate email address with ellipsis

* CXSPA-1611: Remove max-width from mobile view

MarcinGanczorz committed:
Feature/cxspa 1548 add dedicated unit level orders details page (#16426)

* CXSPA-1548: add unit-level order details route, use unit-level order details endpoint

* CXSPA-1548: refactor order-detail-items, introduce wrapper component

* CXSPA-1548: provide order-details-order-entries for unitLevelOrderDetail page

* CXSPA-1548 remove wrapper components, fix dependency issue

* CXSPA-1548: add unit level order overview component

* CXSPA-1548: fix unit order details order entries

* CXSPA-1548: configure cms components

* CXSPA-1548: fix type check, fix tests

* Add license header

* CXSPA-1548: fix link target

Co-authored-by: github-actions[bot] <github-actions[bot]@users.noreply.github.com>

giancorderoortiz committed:
Merge branch 'develop' into epic/unit-level-orders

MarcinGanczorz committed:
CXSPA-1699: fix issue with reducers on unit-level order detail page (#16458)

karolW19 committed:
Feature/cxspa 1534 (#16399)

* CXSPA-1505_CXSPA-1506: add e2e tests for sorting unit-level order history list

* Additional chenges with mocking response

* Adding additional tests regarding checking visibility of an email address in Buyer column

* Adding additional tests regarding checking visibility of an email address in Buyer column and fixing tests

* Adding changes after code review

* Adding changes after code review

* cleanup and polish

* Add license header

Co-authored-by: I565452 <michal.pieprzyca@sap.com>
Co-authored-by: github-actions[bot] <github-actions[bot]@users.noreply.github.com>

I565452 committed:
Feature/cxspa 1534 prettify certain files (#16475)

* CXSPA-1534: prettify some files

* CXSPA-1534: remove unnecessary change

I565452 committed:
feat: cxspa 1534 update tests message and prettify (#16498)

* CXSPA-1534: prettify some files

* CXSPA-1534: remove unnecessary change

* CXSPA-1534: update test messages

KibriyaInamdar committed:
CXSPA-1367: Filtering unit level orders (#16448)

* CXSPA-1367: add filters to the unit-level orders history page

* add filter for buyer and unit

* update filtercontrol name

* add mobile view

* add mobile view

* update mobile view filter

* code refactoring

* add form submit on enter--mobile view

* code refactoring

* code refactoring

* add close and search icon

* add clear function

* update css

* reset buttons and input changes

* refactor styling, indentation

* Clear All and Search go bottom-right of the search inputs

* hide some buttons and inputs for smaller screens and drop a comment

* update css

* fix Clear All icon bug

* mostly bug fixes

* big fixes css

* code refactoring, update navigation

* bug fix

* big fix

* bug fix

* bug fix

* remove unused file

* Add license header

* code refactoring

* code refactoring

* code refactoring add keydown.enter for mobile view

* cleanup some dom and styling

* code refactoring, use formControl

* add filter OnValueChange

* bug fix-- clear all

* code refactoring

* move inline css to scss file

* add remove applied filter button mobile view

* bug fix-css update-- padding-left, padding-right, margin-left, margin-right

* bug fix-lowercase function and variable name, remove subscribe on form submit

* remove formsModule

* fix scrolling bug and slightly change indenting

* bug fix-lowercase function name

* enable scrolling on clearUnitMobile and clearBuyerMobile and fix indenting

* Validations and static code checks, prettify

* specify the keys for the values set in the forms

* prettier

* cover more of the background with the popup

* clear input on pressing the cross button

* fix unit tests

* clear input when clicking the cross button

Co-authored-by: I565452 <michal.pieprzyca@sap.com>
Co-authored-by: Kibriya Inamdar <kibriya.inamdar@sap.com>
Co-authored-by: Andi Braimllari <andi.braimllari@sap.com>
Co-authored-by: github-actions[bot] <github-actions[bot]@users.noreply.github.com>

T0bas-2 committed:
CXSPA-1301: Add B2B order placement logic (#16508)

MarcinGanczorz committed:
feat: CXSPA-1744 remove unit-level orders changes from regular order components (#16506)

* CXSPA-1744: remove unit-level orders changes from regular order components

* CXSPA-1744: add buyer translation for unit level order details

MarcinGanczorz committed:
Merge branch 'develop' into epic/unit-level-orders

MarcinGanczorz committed:
CXSPA-1743: adjust order detail tests to use unitLevelOrderDetails endpoint (#16510)

giancorderoortiz committed:
Merge branch 'develop' into epic/unit-level-orders

giancorderoortiz commented:
Hold merging until all e2e tests pass with CI!

I539321 committed:
change pointed Commerce URL

I565452 committed:
feat: CXSPA-1656 cypress tests for unit-level order history page - filtering and sorting (#16509)

* CXSPA-1656: add e2e tests for filtering on unit-level order history page

* CXSPA-1656: add e2e tests for filtering on unit-level order history page

* CXSPA-1656: update after review comments

* CXSPA-1656: update after review comments

* CXSPA-1656: fix the test for the edge case, when buyers differ only in their email

* CXSPA-1656: remove the test options left unintentionally

I539321 committed:
Feature/test roles and rights (#16533)

* account for the rights as well not just the roles in a cy test

* filter out the unitorderviewergroup in the test

* prettier

szczepaniak-michal commented on .env-cmdrc:
```diff
@@ -1,12 +1,12 @@
 {
   "dev": {
-    "CX_BASE_URL": "https://40.76.109.9:9002"
+    "CX_BASE_URL": "https://20.127.192.33:9002"
```
We should not change default dev server addresses on develop

MarcinGanczorz replied:
This change was requested by @giancorderoortiz until we fix our e2e tests. It should be reverted afterwards.

pavel-ermolin replied:
will be reverted before merging into `develop`

I539321 committed:
Merge branch 'develop' into epic/unit-level-orders

giancorderoortiz committed:
Merge branch 'develop' into epic/unit-level-orders

KibriyaInamdar committed:
Feature/cxspa-1484:add no matching order record text (#16507)

Co-authored-by: Kibriya Inamdar <kibriya.inamdar@sap.com>

T0bas-2 committed:
CXSPA-1301: Create unit-level order history e2e test (#16523)

Co-authored-by: I565452 <michal.pieprzyca@sap.com>

I539321 committed:
Feature/cxspa 1759: Show text with the selected filters (#16514)

* add text that shows the selected filters

* Validations and static code checks

* change when the shown text for the filters is updated

* drop nullifying of the buyerFilterMobileValue variable in clearBuyer

* update the text of the filter values when switching between desktop and mobile

MarcinGanczorz committed:
CXSPA-1805: change the way search icon and reset button is hidden in filter inputs (#16551)

Visibility now rely on the state of the input.

I539321 committed:
revert IP changes

I539321 committed:
Merge branch 'develop' into epic/unit-level-orders

I539321 committed:
change the remove applied filters text

MarcinGanczorz committed:
CXSPA-1882: improve unit test code coverage for unit-level order details implementation (#16558)

KibriyaInamdar committed:
Feature/cxspa 1707 unit test filters (#16538)

* add unit test

* add unit test for clear all and remove applied filters

* unit test for mobile

* add unit test for launch filters and close nav button

* add unit test for clear input

* code refactoring

* change title, add check for mobile view

* add mousedown event for x button click

* update unit test for mobile view

* bug fix remove applied filter test

* fix clear button unit test

* fix unit test desktop view x button

* add constants for literals

* update reference element, use only id

* prettier fix

* update let to const

* update async method to waitForAsync

Co-authored-by: Kibriya Inamdar <kibriya.inamdar@sap.com>

szczepaniak-michal commented on feature-libs/organization/administration/components/user/details/user-details.component.html:
```diff
@@ -38,14 +38,31 @@
+      <div class="property">
+        <label>{{ 'orgUser.rights' | cxTranslate }}</label>
+        <ul class="value">
+          <ng-container *ngFor="let role of model.roles; let isLast = last">
```
Seems as if the last element is not used in this template

I539321 replied:
Hmm yea it seems like we can remove the ` let isLast = last`, can you confirm @T0bas-2?

T0bas-2 replied:
Yes can be removed, also appears in another file without usage, can be removed there too.

szczepaniak-michal commented on feature-libs/organization/administration/components/user/details-cell/user-details-cell.component.ts:
```diff
@@ -6,10 +6,39 @@
 
```
Unit tests are missing for the `UserDetailsCellComponent`

MarcinGanczorz committed:
Merge branch 'develop' into epic/unit-level-orders

# Conflicts:
#	feature-libs/organization/README.md
#	projects/schematics/src/shared/libs-constants.ts
#	projects/storefrontapp/src/app/spartacus/spartacus-features.module.ts
#	projects/storefrontapp/tsconfig.app.prod.json
#	projects/storefrontapp/tsconfig.server.json
#	projects/storefrontapp/tsconfig.server.prod.json
#	tsconfig.compodoc.json
#	tsconfig.json

barteturek committed:
Feature/CXSPA-1312: Update Spartacus e2e tests for edit user component and user roles component (#16478)

Co-authored-by: I561291 <tobias.djermester@sap.com>

KibriyaInamdar committed:
bugfix/cxspa-1881: after search show the search result page (#16560)

* add unit test

* add unit test for clear all and remove applied filters

* unit test for mobile

* add unit test for launch filters and close nav button

* add unit test for clear input

* code refactoring

* change title, add check for mobile view

* add mousedown event for x button click

* update unit test for mobile view

* bug fix remove applied filter test

* fix clear button unit test

* fix unit test desktop view x button

* add constants for literals

* update reference element, use only id

* prettier fix

* update let to const

* mobile search display result page

* update css

* update method on close window

* move css to scss file

* search orders on x button click

* update test

* update input text on close window

* remove type=button for mobile view

* fix unit tests

* update method on close button

Co-authored-by: Kibriya Inamdar <kibriya.inamdar@sap.com>

T0bas-2 committed:
Feature/CXSPA-1958: Refactoring & patch test (#16594)

MarcinGanczorz committed:
CXSPA-1803: change placeholder and aria-labels to fix accessibility issues (#16587)

-add role="search" for unit field

giancorderoortiz committed:
Merge branch 'develop' into epic/unit-level-orders

giancorderoortiz committed:
Merge branch 'develop' into epic/unit-level-orders

giancorderoortiz committed:
Merge branch 'develop' into epic/unit-level-orders

szczepaniak-michal committed:
Merge branch 'develop' into epic/unit-level-orders

giancorderoortiz committed:
Merge branch 'develop' into epic/unit-level-orders

giancorderoortiz committed:
Merge branch 'develop' into epic/unit-level-orders

bgambocjaviniar committed:
fix: Update UnitLevelOrderHistoryFilterComponent unit tests (#16606)

* remove the randomness of test runs

szczepaniak-michal reviewed (changes requested):
Styling changes request from Mikel

morganm58 commented on feature-libs/organization/unit-order/styles/components/unit-level-order-history/_unit-level-order-history.scss:
```diff
@@ -0,0 +1,472 @@
+  .cx-unit-level-order-history-pagination {
+    margin-inline-start: auto;
+    @include media-breakpoint-down(sm) {
+      align-items: flex-start;
```
Pagination should be centered in small screen sizes. The only thing this selector needs is `margin: 0 auto;` everything else should be removed

pavel-ermolin committed:
Merge remote-tracking branch 'origin/develop' into epic/unit-level-orders

bgambocjaviniar committed:
Merge branch 'develop' into epic/unit-level-orders

morganm58 commented on feature-libs/organization/unit-order/styles/components/unit-level-order-history/_unit-level-order-history.scss:
```diff
@@ -0,0 +1,472 @@
+    flex-direction: row;
+
+    .clear-btn {
+      margin-inline-end: 1rem;
```
margin-inline-end should be 2.125rem to match figma mock ups

morganm58 commented on feature-libs/organization/unit-order/styles/components/unit-level-order-history/_unit-level-order-history.scss:
```diff
@@ -0,0 +1,472 @@
+    display: flex;
+    flex-direction: row;
+
+    .clear-btn {
```
This should be `cx-clear-btn`

morganm58 commented on feature-libs/organization/unit-order/components/unit-level-order-history/filter/unit-level-order-history-filter.component.html:
```diff
@@ -0,0 +1,339 @@
+    <button
+      id="closeFilterNavBtn"
+      type="button"
+      class="clear-btn"
```
This should be `cx-clear-btn`

morganm58 commented on feature-libs/organization/unit-order/components/unit-level-order-history/filter/unit-level-order-history-filter.component.html:
```diff
@@ -0,0 +1,339 @@
+    <span>{{ 'unitLevelOrderHistory.buyer' | cxTranslate }}</span>
+    <button
+      type="button"
+      class="clear-btn"
```
This should be `cx-clear-btn`

morganm58 commented on feature-libs/organization/unit-order/components/unit-level-order-history/filter/unit-level-order-history-filter.component.html:
```diff
@@ -0,0 +1,339 @@
+    <span>{{ 'unitLevelOrderHistory.unit' | cxTranslate }}</span>
+    <button
+      type="button"
+      class="clear-btn"
```
This should be `cx-clear-btn`

morganm58 commented on feature-libs/organization/unit-order/components/unit-level-order-history/filter/unit-level-order-history-filter.component.html:
```diff
@@ -0,0 +1,339 @@
+      </button>
+
+      <button
+        class="clear-btn"
```
This should be `cx-clear-btn`

morganm58 commented on feature-libs/organization/unit-order/components/unit-level-order-history/filter/unit-level-order-history-filter.component.html:
```diff
@@ -0,0 +1,339 @@
+
+    <div>
+      <button
+        class="cx-action-link clear-btn"
```
This should be `cx-clear-btn`

morganm58 commented on feature-libs/organization/unit-order/components/unit-level-order-history/filter/unit-level-order-history-filter.component.html:
```diff
@@ -0,0 +1,339 @@
+      </button>
+
+      <button
+        class="clear-btn"
```
This should be `cx-clear-btn`

morganm58 commented on feature-libs/organization/unit-order/components/unit-level-order-history/filter/unit-level-order-history-filter.component.html:
```diff
@@ -0,0 +1,339 @@
+    </div>
+    <div class="cx-unit-level-order-history-filter-nav-button-block">
+      <button
+        class="cx-action-link clear-btn"
```
This should be `cx-clear-btn`

morganm58 commented on feature-libs/organization/unit-order/components/unit-level-order-history/filter/unit-level-order-history-filter.component.html:
```diff
@@ -0,0 +1,339 @@
+    <div class="cx-unit-level-order-history-filter-form-button-block">
+      <button
+        id="clearAllBtn"
+        class="cx-action-link clear-btn"
```
This should be `cx-clear-btn`

morganm58 commented on feature-libs/organization/unit-order/components/unit-level-order-history/filter/unit-level-order-history-filter.component.html:
```diff
@@ -0,0 +1,339 @@
+    <div class="cx-clear-filter">
+      <button
+        id="removeAppliedFiltersBtn"
+        class="cx-action-link clear-btn"
```
This should be `cx-clear-btn`

morganm58 commented on feature-libs/organization/unit-order/styles/components/unit-level-order-history/_unit-level-order-history.scss:
```diff
@@ -0,0 +1,472 @@
+    border-bottom: 2px solid #e0e0e0;
+  }
+
+  .clear-btn {
```
This should be `cx-clear-btn`

morganm58 commented on feature-libs/organization/unit-order/styles/components/unit-level-order-history/_unit-level-order-history.scss:
```diff
@@ -0,0 +1,472 @@
+  }
+
+  .clear-btn {
+    margin-inline-end: 1rem;
```
The margin should be `margin-inline-end: 2.125rem;` to match or be close to the figma mock up

morganm58 commented on feature-libs/organization/unit-order/styles/components/unit-level-order-history/_unit-level-order-history.scss:
```diff
@@ -0,0 +1,472 @@
+
+  .cx-unit-level-order-history-filter-label-wrapper {
+    width: 200px;
+    border: 1px solid lightgray;
```
We should only use core color variables because they are safe for theme switching and pass the accessibility guidelines. This would break if the theme is changed. `lightgray` should be replaced with `var(--cx-color-secondary);` This darker gray matches the figma mock up better and also will pass the accessibility contrast test

morganm58 commented on feature-libs/organization/unit-order/styles/components/unit-level-order-history/_unit-level-order-history.scss:
```diff
@@ -0,0 +1,472 @@
+
+  .cx-unit-level-order-history-filter-reset-button {
+    border: none;
+    color: lightgray;
```
`lightgray` should be replaced with `var(--cx-color-secondary);`

morganm58 commented on feature-libs/organization/unit-order/styles/components/unit-level-order-history/_unit-level-order-history.scss:
```diff
@@ -0,0 +1,472 @@
+  .cx-unit-level-order-history-filter-reset-button {
+    border: none;
+    color: lightgray;
+    background: white;
```
`white` should be replaced with `var(--cx-color-inverse);`

morganm58 commented on feature-libs/organization/unit-order/styles/components/unit-level-order-history/_unit-level-order-history.scss:
```diff
@@ -0,0 +1,472 @@
+    display: block;
+    padding: 10px;
+    padding-top: 15px;
+    color: lightgray;
```
`lightgray` should be replaced with `var(--cx-color-secondary);`

MarcinGanczorz replied:
~~@morganm58  are you sure you want to replace every `lightgray` with `var(--cx-color-secondary);`? cx-color-secondary is a lot darker than lightgray. Maybe we should replace it with `var(--cx-color-light);` instead?~~
Nevermind, I see you want this color to be darker.

morganm58 commented on feature-libs/organization/unit-order/styles/components/unit-level-order-history/_unit-level-order-history.scss:
```diff
@@ -0,0 +1,472 @@
+    }
+
+    .cx-filter-nav-div {
+      box-shadow: 0 4px 2px -2px lightgray;
```
`lightgray` should be replaced with `var(--cx-color-secondary);`

morganm58 commented on feature-libs/organization/unit-order/styles/components/unit-level-order-history/_unit-level-order-history.scss:
```diff
@@ -0,0 +1,472 @@
+      align-content: stretch;
+      background: white;
+      padding-bottom: 0;
+      border: 1px solid lightgray;
```
`lightgray` should be replaced with `var(--cx-color-secondary);`

morganm58 commented on feature-libs/organization/unit-order/styles/components/unit-level-order-history/_unit-level-order-history.scss:
```diff
@@ -0,0 +1,472 @@
+  }
+
+  .cx-unit-level-order-history-filter-input:focus {
+    background: white;
```
`white` should be replaced with `var(--cx-color-inverse);`

morganm58 commented on feature-libs/organization/unit-order/styles/components/unit-level-order-history/_unit-level-order-history.scss:
```diff
@@ -0,0 +1,472 @@
+    label {
+      display: flex;
+      align-content: stretch;
+      background: white;
```
`white` should be replaced with `var(--cx-color-inverse);`

morganm58 commented on feature-libs/organization/unit-order/styles/components/unit-level-order-history/_unit-level-order-history.scss:
```diff
@@ -0,0 +1,472 @@
+    }
+
+    input:focus {
+      background-color: white;
```
`white` should be replaced with `var(--cx-color-inverse);`

morganm58 commented on feature-libs/organization/unit-order/styles/components/unit-level-order-history/_unit-level-order-history.scss:
```diff
@@ -0,0 +1,472 @@
+    }
+
+    input::placeholder {
+      color: lightgray;
```
`lightgray` should be replaced with `var(--cx-color-secondary);`

morganm58 commented on feature-libs/organization/unit-order/components/unit-level-order-detail/unit-level-order-overview/unit-level-order-overview.component.html:
```diff
@@ -0,0 +1,77 @@
+      ></cx-card>
+
+      <cx-card
+        class="display-inline"
```
This class should not be used. If custom styles are needed please create a class using the `cx-name` to name the class. We try to stay away from generic class names like "display-inline"

morganm58 commented on feature-libs/organization/unit-order/styles/components/unit-level-order-overview/_unit-level-order-overview.scss:
```diff
@@ -0,0 +1,61 @@
+          margin-bottom: 0.5rem;
+        }
+
+        .display-inline .cx-card-label-container {
```
This class should not be used. If custom styles are needed please create a class using the `cx-name` to name the class. We try to stay away from generic class names like "display-inline"

morganm58 commented on feature-libs/organization/unit-order/styles/components/unit-level-order-history/_unit-level-order-history.scss:
```diff
@@ -0,0 +1,472 @@
+    border: none;
+    background-color: inherit;
+    font-size: var(--cx-font-size, 1rem);
+    float: right;
```
Please remove the float rule. We replaced floating with using flex-box.

morganm58 commented on feature-libs/organization/unit-order/styles/components/unit-level-order-history/_unit-level-order-history.scss:
```diff
@@ -0,0 +1,472 @@
+  }
+
+  .filter-list-border {
+    border-bottom: 2px solid #e0e0e0;
```
all colors should be variables that are declared in the _variables.scss file
this should be `var(--cx-color-light);`. `filter-list-border` seems like a class that you might reuse on another element. If an element needs a border or styling we should add the styling to it by adding a class if it doesn't have one already. You should remove this and add the border. to the specific html element with a class name following `cx-` naming.

morganm58 commented on feature-libs/organization/unit-order/styles/components/unit-level-order-history/_unit-level-order-history.scss:
```diff
@@ -0,0 +1,472 @@
+    }
+  }
+
+  .cx-vertical-line-separator {
```
`.cx-vertical-line-separator` seems like a class that you might reuse on another element. If an element needs a border or styling we should add the styling to it by adding a class to the element if it doesn't have one already. You should remove this and add the border to the specific html element with a class name following `cx-` naming.

morganm58 reviewed (changes requested):
I've added some comments with changes that should be made.

MarcinGanczorz committed:
CXSPA-2012: fix unit-level orders styling issues (#16630)

* CXSPA-2012: fix unit-level orders styling issues

- replace static colors with cx- variables
- rename css classes, add cx- prefix
- center pagination on mobile view
- change structure of mobile filter view to use flexbox positioning instead of float
- adjust font and icon sizes for mobile filter view

szczepaniak-michal committed:
Merge branch 'develop' into epic/unit-level-orders

MarcinGanczorz committed:
CXSPA-2042: fix sonar issues (#16633)

MarcinGanczorz committed:
CXSPA-2042: fix sonar issue

I539321 committed:
Merge branch 'develop' into epic/unit-level-orders

I539321 committed:
Merge branch 'develop' into epic/unit-level-orders

szczepaniak-michal commented:
Closed due to new pipeline in PR:
https://github.com/SAP/spartacus/pull/16638
